README.md

```diff
@@ -13,57 +13,39 @@ It is also possible to use Reshade Fx shaders.
 ## Disclaimer
 This is one of my first projects ever, so expect it to have bugs. Use it at your own risk.
 
-## Install
-**There are binaries attached to each release, so manually building from source is not required**
+## Building from Source
 
-Once you have the binaries, either by downloading and unpacking them or by following the build instructions below, execute:
-```
-make install
-```
-**Note: do not run this with `sudo`. vkBasalt gets installed for the current user only.**
-
-## Build Dependencies
-
-### Prerequisites
+### Dependencies
 Before building, you will need:
 - GCC >= 9
 - X11 development files
 - glslang
-- spirv-opt
 
-See below on how to install them.
+### Building
+
+**Make sure that the resulting install directory for the .so is found by the linker, otherwise the loader will not find it.** In general, prefer using distro provided packages.
 
-#### Arch-based distributions
-For Arch-based distributions, execute:
-```
-sudo pacman -Syu glslang vulkan-tools lib32-libx11 libx11
-```
-#### Debian/Ubuntu-based distributions
-For newer Debian/Ubuntu-based distributions, execute:
-```
-sudo apt install build-essential gcc-multilib libx11-dev libx11-dev:i386 glslang-tools spirv-tools
 ```
-#### Fedora
-For Fedora, execute:
+git clone https://github.com/DadSchoorse/vkBasalt.git --recurse-submodules
+cd vkBasalt
 ```
-sudo dnf install vulkan-tools glslang libX11-devel glibc-devel.i686 libstdc++-devel.i686 spirv-tools libX11-devel.i686
+
+#### 64bit
+
 ```
-#### Gentoo-based distributions
-For Gentoo-based distributions, execute:
+meson --buildtype=release build.64
```

Comment:
Just a little note here: Meson uses in their docs `builddir` instead of `build` (https://github.com/mesonbuild/meson/commit/f3be687cbbfd1ff25c7ec4ffe8ba4adfee9a6579).
While only an optical change, it highlights better that `meson build` actually doesn't build anything.

I recommend using `builddir` for plain 64 bit (or host arch in that matter), and `builddir.32` for the 32 bit build (which only really applies to amd64 anyway).